### ISSUE TITLE
Using 'r#' breaks syntax highlighting

### ISSUE THREAD

**Issue by depeele** [GrayJack/language-rust-treesitter#3]
With the following snippet, syntax highlighting is fine until the 5th line (`r#type`) after which everything is grey typically all the way to the end of the file. Sometimes, syntax highlighting is restored much further on for reasons I can't ascertain.

```rust
struct MyType{
  id:     u32,
  name:   String,
  value:  u64,
  r#type: String,
  size:   u64,
  total:  u64,
}

```

  
![image](https://user-images.githubusercontent.com/441107/96624536-db848e00-12da-11eb-8524-0afe7ec288ed.png)

**Comment by GrayJack** [comment]
Yes, that sure happen, that is because [tree-sitter-rust](https://github.com/tree-sitter/tree-sitter-rust) either doesn't support it or it has a bug in the parser
